### PR TITLE
Add android_studio package

### PR DESCRIPTION
**Pull request by uberhacker**
Android Studio is the official IDE for Android development, and includes everything you need to build Android apps.  See https://developer.android.com/studio.

Tested on:
- [x] x86_64

### CODE REVIEW

**Comment by skiman6010** [comment]
Man how does this run on lower end Chromebooks?

**Comment by uberhacker** [comment]
If you have a small hdd, you are out of luck.  You can say the same about arm Chromebooks.

File: packages/android_studio.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Android_studio < Package
+  description 'Android Studio is the official IDE for Android development.'
+  homepage 'https://developer.android.com/studio'
+  version '3.3.1.0'
+  source_url 'https://dl.google.com/dl/android/studio/ide-zips/3.3.1.0/android-studio-ide-182.5264788-linux.zip'
```
Comment (JL2210) [inline packages/android_studio.rb]: ... This is not a good idea. You should at least fail out before downloading on other architectures.

Comment (JL2210) [reply packages/android_studio.rb]: http://tools.android.com/build/studio

**Pull request merged**